dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/AbsLimRateLimFeedthroughFreeze.mo

```diff
@@ -0,0 +1,63 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block AbsLimRateLimFeedthroughFreeze "First order filter with absolute and rate limits, and a freezing flag"
```

Comment:
Please explain here in the comment the difference with AbsLimRateLimFirstOrderFreeze

Reply:
It is in fact not a first order but a feedthrough